README.md

```diff
@@ -1,5 +1,6 @@
 # OSPO Project Template
 
+![Release](https://img.shields.io/badge/OSS%20Template-0.3.1-brightgreen.svg)
```

Comment:
Would a different name be more explanatory for the badge? "oss repo template", "repo template" ?

Reply:
Definitely agree, I was going to recommend at minimum including `Template` in the badge display name. Checking to see how other template repositories do it....

Reply:
Just so we're clear, `Release` is the alt text, and the badge itself is ![`OSS Template | 0.3.1`](https://img.shields.io/badge/OSS%20Template-0.3.1-brightgreen.svg). Want to ensure we're all on the same page before I make any changes.

Reply:
`OSS Template` LGTM, but may as well make it consistent between the badge/alt text